bigframes/_config/bigquery_options.py

```diff
@@ -35,6 +35,12 @@
 
 UNKNOWN_LOCATION_MESSAGE = "The location '{location}' is set to an unknown value. Did you mean '{possibility}'?"
 
+LARGE_RESULTS_WARNING_MESSAGE = (
+    "Sampling is disabled because 'allow_large_results' is set to False. "
```

Comment:
Isn't the "head" style sampling still supported, just not sampling by number of bytes?